app/src/main/java/com/devindi/wallpaper/history/HistoryAdapter.kt

```diff
@@ -0,0 +1,58 @@
+package com.devindi.wallpaper.history
+
+import android.net.Uri
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import com.devindi.wallpaper.R
+import com.devindi.wallpaper.model.config.ConfigManager
+import com.devindi.wallpaper.model.history.WallpaperEntry
+import com.devindi.wallpaper.model.map.buildTileRequest
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.history_item.view.*
+import timber.log.Timber
+import java.text.SimpleDateFormat
+
+class HistoryAdapter(private val configManager: ConfigManager): RecyclerView.Adapter<HistoryAdapter.HistoryItemVh>() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/c641b62fa39c60acdf91edee9483647734123681/app/src/main/java/com/devindi/wallpaper/history/HistoryAdapter.kt#L19"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>